x/bank/atlas.toml

```diff
@@ -0,0 +1,34 @@
+[module]
+description = "The bank module is responsible for handling multi-asset coin transfers between accounts and tracking special-case pseudo-transfers which must work differently with particular kinds of accounts."
+homepage = "https://github.com/cosmos/cosmos-sdk"
+keywords = [
+  "tokens",
+  "bank",
+  "transfer",
+  "asset",
+  "fungible",
+  "coins",
+]
+
+name = "x/bank"
+
+[bug_tracker]
+url = "https://github.com/cosmos/cosmos-sdk/issues"
+
+[[authors]]
+name = "alexanderbez"
+
+[[authors]]
+name = "fedekunze"
+
+[[authors]]
+name = "cwgoes"
+
+[[authors]]
+name = "alessio"
```

Comment:
If you want yourself added as an author here, please state so.

Reply:
Hereby I state so :)